### PR TITLE
Actor Definition and Capability Statement consolidation (FHIR-46801, FHIR-46799, FHIR-46814, FHIR-46815, FHIR-46806)

### PR DESCRIPTION
**Pull request by JayMurdoch**
[FHIR-46801](https://jira.hl7.org/browse/FHIR-46801)
* Amend Filler actor definition as described.

[FHIR-46799](https://jira.hl7.org/browse/FHIR-46799)
* Amend Placer actor definition as described.

[FHIR-46814](https://jira.hl7.org/browse/FHIR-46814) & [FHIR-46815](https://jira.hl7.org/browse/FHIR-46815)
* Consolidate sections 6 (Capability Statements) and 9 (Actor Definitions) into one page 'Actors and Capabilities' with links and definitions as described.
* Noting that the Placer, Filler and Patient definitions supplied in 'ActorCapTable' in resolution were outdated, and voted as changed in [FHIR-46799](https://jira.hl7.org/browse/FHIR-46799), [FHIR-46801](https://jira.hl7.org/browse/FHIR-46801) and [FHIR-46802](https://jira.hl7.org/browse/FHIR-46802) - confirmed with @michael-wilson-au.

[FHIR-46806](https://jira.hl7.org/browse/FHIR-46806)
* Changes to text on Conformance page as described.

### CODE REVIEW

File: input/pagecontent/conformance.md [hunk 1]
```diff
@@ -1,18 +1,8 @@
-An [AU eRequesting Placer](ActorDefinition-au-erequesting-actor-placer.html):
+Each AU eRequesting actor as defined in [Actors and Capabilities](capability-statements.html):
 - **SHALL** implement the [mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU eRequesting profiles.
```
Comment (dbojicic) [inline input/pagecontent/conformance.md:2]: This slightly differs from the FHIR-46806 resolution, should say "... elements in the AU eRequesting profiles"

File: input/pagecontent/conformance.md [hunk 2]
```diff
@@ -1,18 +1,8 @@
...
 - **SHALL** implement the required REST API interactions and search parameters as defined in the [AU eRequesting Placer Capability Statement](CapabilityStatement-au-erequesting-placer.html#resourcesSummary1).
```
Comment (dbojicic) [inline input/pagecontent/conformance.md:3]: "as defined in the [AU eRequesting Placer Capability Statement](CapabilityStatement-au-erequesting-placer.html#resourcesSummary1)." should be "as defined in their respective CapabilityStatement."

File: input/pagecontent/conformance.md [hunk 3]
```diff
@@ -1,18 +1,8 @@
...
-- **SHALL** implement the [mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU eRequesting profiles.
-- **SHALL** implement the required REST API interactions and search parameters as defined in the [AU eRequesting Patient Capability Statement](CapabilityStatement-au-erequesting-patient.html#resourcesSummary1).
-
-An [AU eRequesting Server](ActorDefinition-au-erequesting-actor-server.html):
-- **SHALL** implement the [mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU eRequesting profiles.
-- **SHALL** implement the required REST API interactions and search parameters as defined in the [AU eRequesting Server Capability Statement](CapabilityStatement-au-erequesting-server.html#resourcesSummary1).
+Additionally, the [AU eRequesting Server](ActorDefinition-au-erequesting-actor-server.html) Actor:
 - **SHALL** specify the implemented resource types, associated AU eRequesting profiles, REST API interactions and search parameters in the server's CapabilityStatement.
```
Comment (dbojicic) [inline input/pagecontent/conformance.md:6]: This is slightly different to FHIR-46806 resolution description, please update to match.

File: input/pagecontent/conformance.md [hunk 4]
```diff
@@ -1,18 +1,8 @@
...
 - **SHOULD** [declare conformance](#declaring-conformance) to the [AU eRequesting Server Capability Statement](CapabilityStatement-au-erequesting-server.html) in the server's CapabilityStatement.
```
Comment (dbojicic) [inline input/pagecontent/conformance.md:7]: This is slightly different to FHIR-46806 resolution description, please update to match.

File: input/pagecontent/conformance.md [hunk 5]
```diff
@@ -37,10 +27,12 @@ The [Profiles and Extensions](profiles-and-extensions.html) page lists the AU eR
 
 AU eRequesting profile elements include mandatory and *Must Support* requirements. [Mandatory elements](#mandatory-elements) are required and have a minimum cardinality of 1 (min=1). [Must Support](#must-support-and-obligation) elements have defined conformance obligations in AU eRequesting based on actor roles.
 
+The [Actors and Capabilities](capability-statements.html) page list the AU eRequesting Actor definitions and the CapabilityStatements.
```
Comment (dbojicic) [inline input/pagecontent/conformance.md:30]: "the CapabilityStatements" should be 'their CapabilityStatements"

File: input/pagecontent/index.md [hunk 6]
```diff
@@ -90,8 +68,7 @@ This guide is divided into several pages which are listed at the top of each pag
...
   - [Profiles and Extensions](profiles-and-extensions.html): This page describes the profiles and extensions that are defined in this guide to support electronic requesting. Each profile page includes a narrative description and guidance, formal definition and a "Notes" section which summarises the supported search transactions for each profile. Although the guidance typically focuses on the profiled elements, it may also may focus on un-profiled elements to aid with implementation.
   - [Terminology](terminology.html): This page lists the value sets and code systems supported in this guide.
-  - [Capability Statements](capability-statements.html): This page defines the expected FHIR capabilities of AU eRequesting Server and AU eRequesting Placer.
-  - [Actor Definitions](actors.html): This page defines the AU aRequesting actors, AU eRequesting Placer, AU eRequesting Filler and AU eRequesting Patient Access.
+  - [Actors and Capabilities](capability-statements.html): This page defines the expected FHIR capabilities of AU eRequesting Server and AU eRequesting Placer.
```
Comment (dbojicic) [inline input/pagecontent/index.md:71]: The description needs to be updated to reflect the consolidation of content from the previous Capability Statements and Actor Definitions pages.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments

File: input/pagecontent/conformance.md (hunk 1 above)
Comment (JayMurdoch) [reply input/pagecontent/conformance.md:2]: Amended, thanks.

File: input/pagecontent/conformance.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/conformance.md:3]: Amended, thanks.

File: input/pagecontent/conformance.md (hunk 4 above)
Comment (JayMurdoch) [reply input/pagecontent/conformance.md:7]: Amended, thanks.

File: input/pagecontent/conformance.md (hunk 5 above)
Comment (JayMurdoch) [reply input/pagecontent/conformance.md:30]: Amended, thanks.

File: input/pagecontent/index.md (hunk 6 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md:71]: Amended, thanks.

File: input/pagecontent/conformance.md [hunk 7]
```diff
@@ -37,10 +27,12 @@ The [Profiles and Extensions](profiles-and-extensions.html) page lists the AU eR
 
 AU eRequesting profile elements include mandatory and *Must Support* requirements. [Mandatory elements](#mandatory-elements) are required and have a minimum cardinality of 1 (min=1). [Must Support](#must-support-and-obligation) elements have defined conformance obligations in AU eRequesting based on actor roles.
 
+The [Actors and Capabilities](capability-statements.html) page list the AU eRequesting Actor definitions and their CapabilityStatements.
```
Comment (dbojicic) [inline input/pagecontent/conformance.md:30]: It seems that the resolution description for section 2.1.1 AU eRequesting Profiles and Extensions is sort of overwritten by [FHIR-46812](https://jira.hl7.org/browse/FHIR-46812), as the Profiles and Extensions page now lists not only the AU eRequesting extensions defined for this implementation guide but also referenced extensions.

**Review by dbojicic — CHANGES_REQUESTED** [review]
One minor comment.

File: input/pagecontent/conformance.md (hunk 7 above)
Comment (michael-wilson-au) [reply input/pagecontent/conformance.md:30]: yes we should also make that update related to FHIR-46812 in this branch

Comment (JayMurdoch) [reply input/pagecontent/conformance.md:30]: Amended, thanks.

**Review by michael-wilson-au — APPROVED** [review]

**Pull request merged**